tests/core/models/test_escn_compiles.py

```diff
@@ -70,8 +70,8 @@ def load_model(type: str, compile=False, export=False):
             cutoff=CUTOFF,
             max_num_elements=MAX_ELEMENTS,
             num_layers=8,
-            lmax_list=[4],
-            mmax_list=[2],
+            lmax_list=[6],
```

Comment:
maybe leave a note here to mention some additional things we can add to make this test more sensitive: add backwards, add better weight initialization etc